orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -4803,6 +4803,7 @@ def forward(self, a):
     del os.environ["ORTMODULE_SKIPCHECK_POLICY"]
 
 
+@pytest.mark.skipif(Version(torch.__version__) < Version("1.13.0"), reason="PyTorch 1.12 incompatible")
```

Comment:
This skip is for the deprecation warning right? For torch >= 1.13.0, is the name collision check still valid?

Reply:
This skip is for the new additional torch warning after torch 1.13. for torch <1.13, the name collision check is valid , but the len(warning_record) should equals to 2.